src/Lachain.Core/Blockchain/VM/ExternalHandler.cs

```diff
@@ -316,6 +323,17 @@ public static void Handler_Env_GetAddress(int resultOffset)
             var result = (frame.CurrentAddress).ToBytes();
             SafeCopyToMemory(frame.Memory, result, resultOffset);
         }
+        
+        public static void Handler_Env_GetMsgValue(int dataOffset)
+        {
+            Logger.LogInformation($"Handler_Env_GetMsgValue({dataOffset})");
```

Comment:
Does it need to be here? I think, it's better to either remove it or add logging to other handlers as well

Reply:
We can have this right now and eventually update the other handlers with logging as well